### ISSUE TITLE
Replay is instantly jumping to another time point

### ISSUE THREAD

**Issue by akiya-nagatsuka** [ppy/osu#25146]
### Type

Game behaviour

### Bug description

1. Open any replay
2. Go a bit through it
3. Press pause
2. Hold left or right key (the replay is keeping to be paused)
3. Press Resume -> The map replay is instantly jumping to another time point depends on the previous step.

### Version

2023.1008.1

### Logs

[database.log](https://github.com/ppy/osu/files/12918954/database.log)
[input.log](https://github.com/ppy/osu/files/12918955/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/12918956/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/12918957/network.log)
[performance.log](https://github.com/ppy/osu/files/12918958/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/12918959/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/12918960/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/12918961/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/12918962/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/12918963/runtime.log)
[updater.log](https://github.com/ppy/osu/files/12918964/updater.log)

**Comment by akiya-nagatsuka** [comment]
Both solutions looks good for me
1. Show on the screen that the replay current time point is actually moving.
2. Don't move the current replay time point implicitly, so after pressing Resume the replay continues from the same time point.

**Comment by akiya-nagatsuka** [comment]
@peppy this is my first preposition (actually preferable in my opinion). 
When you try to seek in a paused state you should see some feedback from the game. So unpause and scroll the replay explicitly should be the best idea. The only thing I am not really sure: should we pause again or resume the replay when the user releases the seek button, previously pressed pause :thinking:

**Comment by peppy** [comment]
Yeah, making things work like a tape record is the end goal. My hope is to actually make audio pitch up and down with the seek, but implementing this is a rather low priority task.

**Comment by Joehuu** [comment]
Probably resolved in https://github.com/ppy/osu/pull/26606.
